boto/vpc/vpc.py

```diff
@@ -91,17 +94,21 @@ def __init__(self, connection=None):
         self.vpc_id = None
         self.enable_dns_hostnames = None
         self.enable_dns_support = None
+        self.description = None
         self._current_attr = None
 
     def startElement(self, name, attrs, connection):
-        if name in ('enableDnsHostnames', 'enableDnsSupport'):
+        if name in ('enableDnsHostnames', 'enableDnsSupport',
+                    'description'):
             self._current_attr = name
 
     def endElement(self, name, value, connection):
         if name == 'vpcId':
             self.vpc_id = value
         elif name == 'value':
-            if value == 'true':
+            if self._current_attr == 'description':
```

Comment:
Whouldn't it better to use parsing like this: https://github.com/C2Devel/boto/blob/master/boto/ec2/volume.py#L104
CC: @gmmephisto